scout/parse/panel.py

```diff
@@ -286,7 +285,10 @@ def parse_panel_app_gene(app_gene, hgnc_map):
 
     hgnc_symbol = app_gene["GeneSymbol"]
     # Returns a set of hgnc ids
-    hgnc_ids = get_correct_ids(hgnc_symbol, hgnc_map)
+    ensembl_ids = app_gene["EnsembleGeneIds"]
```

Comment:
This could possibly warrant a comment somewhere - perhaps in the docstring to not clutter the code - stating:
that sic! PanelApp names Ensembl Gene ID EnsembleGeneIds.

Reply:
done